docs/ref/request-response.txt

```diff
@@ -1091,8 +1091,10 @@ Attributes
     optimized for binary files. It uses `wsgi.file_wrapper`_ if provided by the
     wsgi server, otherwise it streams the file out in small chunks.
 
-    If ``as_attachment=True``, the ``Content-Disposition`` header is set, which
-    asks the browser to offer the file to the user as a download.
+    If ``as_attachment=True``, the ``Content-Disposition`` header is set to
+    ``attachment``, which asks the browser to offer the file to the user as a
+    download. If it's ``False``, ``Content-Disposition`` is set to ``inline``,
+    which tells the browser to handle it in the way that it's configured to.
```

Comment:
This seems a bit clunky. Maybe something like: 

> Otherwise a default `inline` `Content-Disposition` header will only be set if a `filename` is available. 

(This assuming we **don't** always set it, as per other comment.) 

**Either way**, having discussed `Content-Disposition` here, do we not need to adjust the `...headers are automatically set...` sentence below to remove it from there?

Reply:
Changed to suggested sentence.

> Either way, having discussed `Content-Disposition` here, do we not need to adjust the `...headers are automatically set...` sentence below to remove it from there?

Yeah, I agree. Removed it.